code-for-canada/project-thundercat
# [Markdown] Storing markdown files/content in DB

# Description

This PR is introducing some back end and front end changes. 

**Back end changes:**
I created a new migration called _0011_upload_background_markdown.py_, that creates the needed items, item types and items text in order to store the markdown content in the databse.

**====> Note that we'll need to find another way to insert this type of data when we'll need to upload the protected test background content <====**

Also, the concerned view as been updated in order to get the needed data and tests have been added.

**Front end changes:**
I connected the background sections (background information, organizational information, organizational structure and team information) with markdown files/content from the database.

I also deleted all the local markdown files.

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

**Background Information (EN):**

![image](https://user-images.githubusercontent.com/23021242/60269800-2516c180-98bd-11e9-8224-f78f368f34bf.png)

**Background Information (FR):**

![image](https://user-images.githubusercontent.com/23021242/60269813-2b0ca280-98bd-11e9-9b5a-d6adff846dfa.png)

**Organizational Information (EN):**

![image](https://user-images.githubusercontent.com/23021242/60269847-3b248200-98bd-11e9-961d-8b0deea5e17f.png)

**Organizational Information (FR):**

![image](https://user-images.githubusercontent.com/23021242/60269861-437cbd00-98bd-11e9-879d-8be245016e98.png)

**Organizational Structure (EN):**

![image](https://user-images.githubusercontent.com/23021242/60269901-53949c80-98bd-11e9-9f57-293656fddc5d.png)

**Organizational Structure (FR):**

![image](https://user-images.githubusercontent.com/23021242/60269910-598a7d80-98bd-11e9-97a2-f06288d3d703.png)

**Team Information (EN):**

![image](https://user-images.githubusercontent.com/23021242/60269955-6d35e400-98bd-11e9-898b-70bc572864a8.png)

![image](https://user-images.githubusercontent.com/23021242/60269974-74f58880-98bd-11e9-9d39-e8e66e8400aa.png)

**Team Information (FR):**

![image](https://user-images.githubusercontent.com/23021242/60269991-7e7ef080-98bd-11e9-93e5-098180be1940.png)

![image](https://user-images.githubusercontent.com/23021242/60270008-8474d180-98bd-11e9-891f-24b0329dd483.png)

# Testing

Manual steps to reproduce this functionality:

1.  Launch the eMIB Sample Test.
2.  Open the _Background_ tab.
3.  Navigate on each sections using the side menu and make sure that the content is right and well formatted in both languages.

# Checklist
